src/towncrier/build.py

```diff
@@ -258,7 +258,8 @@ def __main(
             "What is seen below is what would be written.\n",
             err=to_err,
         )
-        click.echo(content)
+        # output as bytes to prevent universal_newlines modifying content #453
```

Comment:
I prefer to have a self contained comment, that explains why this is needed without having to search the internet / GitHub :)

I still don't fully understand what went wrong here and what is the best way to fix it..

As mentioned by Chris, I think that a better idea is to fix this inside `render_fragments`